components/asio/idf_component.yml

```diff
@@ -1,6 +1,9 @@
-version: "1.28.0~0"
-description: ASIO
+version: "1.28.0~1"
```

Comment:
Are we missing the commitizen file update?

Reply:
Good catch! thanks, mustve been an auto-merge issue that I missed